geomesa-accumulo/geomesa-accumulo-datastore/src/test/resources/process/join-bin.xml

```diff
@@ -0,0 +1,102 @@
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+    xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsAll.xsd">
+  <ows:Identifier>geomesa:Join</ows:Identifier>
+  <wps:DataInputs>
+    <wps:Input>
+      <ows:Identifier>unique</ows:Identifier>
+      <wps:Reference mimeType="text/xml" xlink:href="http://geoserver/wps" method="POST">
+        <wps:Body>
+          <wps:Execute version="1.0.0" service="WPS">
+            <ows:Identifier>geomesa:Unique</ows:Identifier>
+            <wps:DataInputs>
+              <wps:Input>
+                <ows:Identifier>features</ows:Identifier>
+                <wps:Reference mimeType="text/xml" xlink:href="http://geoserver/wfs" method="POST">
+                  <wps:Body>
+                    <wfs:GetFeature service="WFS" version="1.0.0" outputFormat="GML2" xmlns:geomesa="geomesa">
+                      <wfs:Query typeName="geomesa:main"/>
+                    </wfs:GetFeature>
+                  </wps:Body>
+                </wps:Reference>
+              </wps:Input>
+              <wps:Input>
+                <ows:Identifier>attribute</ows:Identifier>
+                <wps:Data>
+                  <wps:LiteralData>pk</wps:LiteralData>
+                </wps:Data>
+              </wps:Input>
+              <wps:Input>
+                <ows:Identifier>filter</ows:Identifier>
+                <wps:Data>
+                  <wps:ComplexData mimeType="text/plain; subtype=cql"><![CDATA[age = 5]]></wps:ComplexData>
+                </wps:Data>
+              </wps:Input>
+              <wps:Input>
+                <ows:Identifier>histogram</ows:Identifier>
+                <wps:Data>
+                  <wps:LiteralData>false</wps:LiteralData>
+                </wps:Data>
+              </wps:Input>
+            </wps:DataInputs>
+            <wps:ResponseForm>
+              <wps:RawDataOutput mimeType="text/xml; subtype=wfs-collection/1.0">
+                <ows:Identifier>result</ows:Identifier>
+              </wps:RawDataOutput>
+            </wps:ResponseForm>
+          </wps:Execute>
+        </wps:Body>
+      </wps:Reference>
+    </wps:Input>
+    <wps:Input>
+      <ows:Identifier>join</ows:Identifier>
+      <wps:Reference mimeType="text/xml" xlink:href="http://geoserver/wfs" method="POST">
+        <wps:Body>
+          <wfs:GetFeature service="WFS" version="1.0.0" outputFormat="GML2" xmlns:geomesa="geomesa">
+            <wfs:Query typeName="geomesa:join"/>
+          </wfs:GetFeature>
+        </wps:Body>
+      </wps:Reference>
+    </wps:Input>
+    <wps:Input>
+      <ows:Identifier>attribute</ows:Identifier>
+      <wps:Data>
+        <wps:LiteralData>pk</wps:LiteralData>
+      </wps:Data>
+    </wps:Input>
+    <wps:Input>
+      <ows:Identifier>bins</ows:Identifier>
+      <wps:Data>
+        <wps:LiteralData>true</wps:LiteralData>
+      </wps:Data>
+    </wps:Input>
+    <!--<wps:Input>
```

Comment:
I left them in there as examples of the possible input configuration available. If they're not included, it will try to use sensible defaults.